src/Zicht/Bundle/UrlBundle/Aliasing/Listener.php

```diff
@@ -21,11 +21,26 @@
  */
 class Listener
 {
+    const SLASH_SUFFIX_IGNORE = 'ignore-404';
```

Comment:
I'm still not very happy with the naming here. "Ignore" might suggest the behaviour of "accept", which is to also take the _wrong_ URL (with `/` suffix) and just continues to render a response. "ignore" is how it is now, trying to find a URL alias **including** that slash, which isn't there most likely, so the listener does nothing and finally resulting in a 404 error.

Any suggestions for not doing anything with a possible slash suffix (and let it resolve by other means)?

- "deflect"?
- "no-action"?
- "pass"?
- `null`?
- "none"?